templates/ecommerce/order/email.html

```diff
@@ -31,7 +31,7 @@
         <td style="border-color:#E4E4E4;padding:10px;text-align:left">
           <a href="{% full_url 'product' position.product_id %}"
              style="text-decoration: none"><!--
-             -->{{ position.name }} {{ position.mark }}
+             -->{{ position.catalog_name }}
```

Comment:
`Position` model doesn't have the `catalog_name` property and `mark` field too. Let's create a todo for this bug

Reply:
it's not about stb repo, so created it manually
https://github.com/fidals/refarm-site/issues/322

Reply:
@duker33 Right now `Product.mark` exist only in STB, but not in SE, so why not about this repo?

Reply:
@artemiy312 , sorry, i'll create pdd subtask